### ISSUE TITLE
Add docs on how to build `ga_IE.dic`

### ISSUE THREAD

**Issue by wooorm** [kscanne/gaelspell#2]
(Or alternative title: Use GitHub Releases for hunspell)

👋

I package hunspell dictionaries for use in the JS ecosystem: [`wooorm/dictionaries`](https://github.com/wooorm/dictionaries), and I can’t seem to build this project:

```sh
$ make ga_IE.aff ga_IE.dic
cat myspell-header hunspell-header > myspelltemp.txt
./ispellaff2myspell --charset=latin1 gaeilge.aff --myheader myspelltemp.txt | sed 's/""/0/' | sed '40,$s/"//g' | perl -p -e 's/^PFX S( +)([a-z])( +)[a-z]h( +)[a-z](.*)/print "PFX S$1$2$3$2h$4$2$5\nPFX S$1\u$2$3\u$2h$4\u$2$5";/e' | sed 's/S Y 9$/S Y 18/' | sed 's/\([]A-Z]\)1$/\1/' > ga_IE.aff
rm -f myspelltemp.txt
make: *** No rule to make target `/Users/tilde/math/code/data/Dictionary/IG', needed by `gaeilge.raw'.  Stop.
```

(related to https://github.com/kscanne/gaelspell/commit/19ba3b4fdfb6b4b3fba91bd62ef28bd6ea334c73)

Is there a way for other people to build this file, or alternatively: is it possible to include hunspell files in the releases assets?

**Comment by kscanne** [comment]
Thanks for packaging the Irish dictionary! I'll probably just do a hunspell release here on github. Will drop a note here when I do (this coming week I hope).

**Comment by wooorm** [comment]
Sweet, thanks Kevin!

**Comment by kscanne** [comment]
Sorry for the delay, I decided I'd take the opportunity to do a new release (v5.1) and I've included a file hunspell-ga-5.1.zip which has the latest .dic and .aff files. Will this work for you?

**Comment by wooorm** [comment]
Yeah, sure! Thanks!
I’ll be building the dictionaries in `wooorm/dictionaries` every once in a while, picking up the latest releases to include the new changes. So it def works!
